app/models/person.rb

```diff
@@ -172,6 +172,10 @@ def self.search_order
     }.call
   end
 
+  def self.people_names(people)
```

Comment:
Well, I like to have it as a separate method, if possible. I don't know of a better place for that though. Any ideas?